packages/frontend/src/pages/definitions/getProps.tsx

```diff
@@ -0,0 +1,67 @@
+import { Cheerio, Element,load } from 'cheerio'
+import fsx from 'fs-extra'
+import MarkdownIt from 'markdown-it'
+import path from 'path'
+
+import { Config } from '../../build/config'
+import { getFooterProps, getNavbarProps } from '../../components'
+import { Wrapped } from '../Page'
+import { DefinitionsPageProps } from './DefinitionsPage'
+import { renderHeading } from './renderHeading'
+
+export function getProps(config: Config): Wrapped<DefinitionsPageProps> {
+  return {
+    props: {
+      navbar: getNavbarProps(config, 'definitions'),
+      title: 'Definitions',
+      ratings: getHtml(),
+      footer: getFooterProps(config),
+    },
+    wrapper: {
+      metadata: {
+        title: 'L2BEAT - Definitions',
+        description:
+          'Definitions of rollup ratings used on L2BEAT. Learn more about the L2BEAT rollup ratings and how they are calculated.',
+        image: 'https://l2beat.com/meta-images/overview-scaling.png',
+        url: 'https://l2beat.com/faq/',
+      },
+    },
+  }
+}
+
+function getHtml() {
```

Comment:
Is this copied from faq? We should share the code instead

Reply:
@sz-piotr Because of https://linear.app/l2beat/issue/L2B-815/clean-up-the-faq-subpage I decided to leave the FAQ page as is rn.